src/actions/users.js

```diff
@@ -504,6 +504,44 @@ export function getProfilesInChannel(channelId: string, page: number, perPage: n
     };
 }
 
+export function getProfilesInGroupChannels(channelsIds: Array<string>): ActionFunc {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        const {currentUserId} = getState().entities.users;
+
+        let channelProfiles = null;
+        try {
+            channelProfiles = await Client4.getProfilesInGroupChannels(channelsIds.slice(0, General.MAX_GROUP_CHANNELS_FOR_PROFILES));
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch, getState);
+            dispatch(logError(error));
+            return {error};
+        }
+
+        const actions = [];
+        for (const channelId in channelProfiles) {
+            if (channelProfiles.hasOwnProperty(channelId)) {
+                const profiles = channelProfiles[channelId];
+
+                actions.push(
+                    {
+                        type: UserTypes.RECEIVED_PROFILES_LIST_IN_CHANNEL,
+                        data: profiles,
+                        id: channelId,
+                    },
+                    {
+                        type: UserTypes.RECEIVED_PROFILES_LIST,
+                        data: removeUserFromList(currentUserId, [...profiles]),
+                    },
+                );
+            }
+        }
+
+        dispatch(batchActions(actions), getState);
```

Comment:
No need to pass `getState` to dispatch

```suggestion
        dispatch(batchActions(actions));
```

Reply:
Done!